code/game/gamemodes/clock_cult/clock_structures/traps/brass_skewer.dm

```diff
@@ -48,16 +48,24 @@
 /obj/structure/destructible/clockwork/trap/brass_skewer/activate()
 	if(density)
 		return
-	var/mob/living/carbon/squirrel = locate() in get_turf(src)
+	var/mob/living/squirrel = locate() in get_turf(src)
 	if(squirrel)
-		squirrel.visible_message("<span class='boldwarning'>A massive brass spike erupts from the ground, impaling [squirrel]!</span>", \
-		"<span class='userdanger'>A massive brass spike rams through your chest, hoisting you into the air!</span>")
-		squirrel.emote("scream")
-		playsound(squirrel, 'sound/effects/splat.ogg', 50, TRUE)
-		playsound(squirrel, 'sound/misc/desceration-03.ogg', 50, TRUE)
-		squirrel.apply_damage(20, BRUTE, "chest")
-		mouse_opacity = MOUSE_OPACITY_OPAQUE //So players can interact with the tile it's on to pull them off
-		buckle_mob(squirrel, TRUE)
+		if(iscyborg(squirrel))
+			if(!squirrel.stat)
+				squirrel.visible_message("<span class='boldwarning'>A massive brass spike erupts from the ground, rending [squirrel]'s chassis but shattering into pieces!</span>", \
+				"<span class='userdanger'>A massive brass spike rips through your chassis and bursts into shrapnel in your casing!</span>")
+				squirrel.adjustBruteLoss(50)
+				squirrel.Stun(20)
+				addtimer(CALLBACK(src, .proc/take_damage, max_integrity), 1)
```

Comment:
the spike should actually go through the borg for a very brief moment before this timer happens.

Reply:
I wasn't sure fluffwise if the spear was powerful enough to pierce an entire metal machine, but I suppose it could go either way.